api/s3/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
+/Cargo.lock
```

Comment:
This sub-crate is in Cargo workspace, so why this `.gitignore` is required?

Reply:
It is not, forgot to clean it up.